demo/admin/src/common/ContentScopeProvider.tsx

```diff
@@ -35,7 +36,12 @@ export const ContentScopeProvider = ({ children }: Pick<ContentScopeProviderProp
     }));
 
     if (user.allowedContentScopes.length === 0) {
-        throw new Error("User does not have access to any scopes.");
+        return (
+            <>
+                Error: user does not have access to any scopes.
```

Comment:
We should mask this with FormattedMessage if customers can use impersonation in the future.